### DISCUSSION TITLE
macOS version can browse compressed file

### DISCUSSION THREAD

**Discussion by DecoAri** [aonez/Keka#1461] [Ideas]
Hello, Keka iOS & iPadOS can browse files in compression. But macOS version cannot. So can developer make macOS version has browse function?

**Comment by gingerbeardman** [comment]
Work on this is in progress, see #100 

Until then, if you have an M-series Mac, you can use the iOS version of Keka on macOS

**Comment by DecoAri** [comment]
Oh yes. I have tried iOS version. The UI is so small. That's feel not so great. lol. Looking forward to macOS version haha.

**Comment by aonez** [comment]
Can you share some screenshots with some feedback about what you feel is
too small?

**Reply by DecoAri** [reply]
Oh. This is a personal feeling. Maybe other people do not have same feeling.

**Comment by aonez** [comment]
Sure! Just curious, maybe I can improve it for macOS while the full version
is not ready.

**Comment by MurtadhaInit** [comment]
Really looking forward to that feature on MacOS, it would make the app perfect for me.
